src/main/java/teammates/logic/backdoor/BackDoorLogic.java

```diff
@@ -455,7 +455,8 @@ private void injectRealIdsIntoResponseComments(Collection<FeedbackResponseCommen
             String questionKey = makeQuestionKey(sessionKey, questionNumber);
             comment.feedbackQuestionId = questionIdMap.get(questionKey);
 
-            String[] responseIdParam = comment.feedbackResponseId.split("%");
+            // format of feedbackResponseId: questionNumber%giverEmail%recipient
+            String[] responseIdParam = comment.feedbackResponseId.split("%", 3);
```

Comment:
If we don't add 3, for id like `1%test@tmt.test%%GENERAL%`, it will give incorrect split.